src/main/CommandHandler.cpp

```diff
@@ -965,4 +968,18 @@ CommandHandler::maintenance(std::string const& params, std::string& retStr)
         retStr = "No work performed";
     }
 }
+
+void
+CommandHandler::clearMetrics(std::string const& params, std::string& retStr)
+{
+    std::map<std::string, std::string> map;
+    http::server::server::parseParams(params, map);
+
+    std::string domain;
+    maybeParseNumParam(map, "domain", domain);
```

Comment:
same comment than in a different PR, `maybeParseNumParam` needs to be renamed if we're using it to parse types other than numbers.

Reply:
The other PR was updated with the name change. for this PR I was going off of master. Depending on what gets merged first, I'll be able to update it during rebase.